api/src/routes/docs/mod.rs

```diff
@@ -66,8 +66,11 @@ pub async fn handle_summary(cache: Cache) -> Result<impl Reply, Rejection> {
 
     let summary = response.text().await.map_err(|_| warp::reject())?;
 
-    let html =
-        DocsParser::new(&summary, url::Url::from_str("http://localhost:3000/docs").unwrap()).parse();
+    let html = DocsParser::new(
+        &summary,
+        url::Url::from_str("http://localhost:3000/docs/").unwrap(),
```

Comment:
http://localhost:3000/docs/ <- Gets count as file path without slash at end (at least I think it does)